### ISSUE TITLE
Crashes when opening shops

### ISSUE THREAD

**Issue by oldjollysanta** [glisco03/numismatic-overhaul#62]
[crash-2022-12-04_13.01.14-client.txt](https://github.com/wisp-forest/numismatic-overhaul/files/10147777/crash-2022-12-04_13.01.14-client.txt)

Numismatic overhaul version 0.2.5
Minecraft 1.19.2

Crashes when trying to activate a shop. Wasn't happening on older version, but I recently updated a bunch of mods (including this one)
[also I noticed a similar bug reported, but I don't have Charm installed]

**Comment by Noaaan** [comment]
I believe this is could be caused by owo-lib 0.9.0+, which changed a deprecated function, causing this to crash. 
For now I recommend downgrading, but we are aware of this and will get it fixed soon.

**Comment by oldjollysanta** [comment]
Thanks, I'll test that tonight when I update the server

**Comment by oldjollysanta** [comment]
owo 8.5 did the trick. 

Side note:
I was against updating the server (downgrading the mod on the server) as it is already a hassle for most to do it so often (as I recently just updated it and caused this issue), so I'll push it to the next planned update. 
But I tested downgrading client end and it works both single player and also on my server that's using 0.9.+ owo. 
I'll just disable shops for now until my next planned update.
